### DISCUSSION TITLE
No Crop Feature Visible

### DISCUSSION THREAD

**Discussion by HonestAbe7** [FossifyOrg/General-Discussion#408] [Q&A]
I can't find the "crop" feature for photos. This forces me to use the stock phone editor. Am I missing where this is since this app says this is possible? Thanks

**Comment by Aga-C** [comment]
There is a crop function:
1. Open your photo.
2. Tap the pencil icon at the bottom.
3. Choose Fossify Gallery's editor.
4. Crop is the central icon on the bottom action bar (rectangle with two markings in the corners). It's the default tool after opening the editor.

**Reply by HonestAbe7** [reply]
Thank you for your  detailed reply. I now know.
